docs/conf.py

```diff
@@ -61,7 +62,7 @@
 
 # General information about the project.
 project = u'PTSA'
-copyright = u'2017'
+copyright = str(datetime.now().year)
```

Comment:
We should probably change this to 2018 and update whenever we do edits.

Reply:
That's exactly what this does...